kubernetes/deployments_kubeclient_blackbox_test.go

```diff
@@ -714,6 +714,7 @@ func TestConfigMapEnvironments(t *testing.T) {
 			_, err := kubernetes.NewKubeClient(config)
 			if testCase.shouldFail {
 				require.Error(t, err, "Expected an error")
+				require.True(t, fixture.metrics.closed, "Metrics not closed after error")
```

Comment:
Just a suggestion: messages in test should indicate the desired outcome.  E.g. "Metrics must be closed after error".

Reply:
Good suggestion. I've changed this message.